lab2-extended-kalman-filter/simulation.m

```diff
@@ -38,16 +38,10 @@
 %% Simulation
 for t=2:length(T)
     x_ideal(:,t) = x_ideal(:,t-1) + v(:)*dt;
+    x(:,t) = x(:,t-1) + mvnrnd(zeros(1,n),R,1)';

```

Comment:
previously you were using Q here (instead of R)
also, I changed it so its `zeros(1,n)` instead of `zeros(n,1)` because MU needed to be a row vector